README.md

```diff
@@ -228,236 +250,52 @@ messages about the conversion.
 
 ## Example Usage
 
-The following examples assume ``harbourbridge`` has been added to your PATH
-environment variable.
-
-To use HarbourBridge on a PostgreSQL database called mydb, run:
-
-```sh
-pg_dump mydb | harbourbridge
-```
-
-The tool can also be applied to an existing pg_dump file:
-
-```sh
-harbourbridge < my_pg_dump_file
-```
-
-To specify a particular Spanner instance to use, run:
-
-```sh
-pg_dump mydb | harbourbridge -instance my-spanner-instance
-```
-
-By default, HarbourBridge will generate a new Spanner database name to populate.
-You can override this and specify the database name to use by:
+Driver types (source DB) that are supported by Harbourbridge:
```

Comment:
Suggest deleting the paragraph -- I don't see what it adds here, and it kind of repeats the list from a few lines above.

Driver types (source DB) that are supported by Harbourbridge:
1. postgres
2. pgdump
3. mysql
4. mysqldump

Reply:
Just added it for better clarity, if directly referring to `example usage` section. But yeah sure agreed, it's kind of redundant. I will delete this paragraph.